lib/ResolverFactory.js

```diff
@@ -124,11 +144,11 @@ function createOptions(options) {
 			typeof options.cacheWithContext !== "undefined"
 				? options.cacheWithContext
 				: true,
-		descriptionFiles: options.descriptionFiles || ["package.json"],
+		descriptionFiles: Array.from(
```

Comment:
we can do `Array.from` later on line 206 to keep here consistent Set options, but I decided to keep optimization logic in normalization.